cli_tools/gce_vm_image_import/importer/bootable_disk_processor.go

```diff
@@ -16,37 +16,103 @@ package importer
 
 import (
 	"context"
+	"fmt"
+	"log"
 	"os"
 	"path"
 	"strconv"
 	"strings"
 
-	"github.com/GoogleCloudPlatform/compute-image-tools/daisy"
-
+	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/common/disk"
 	daisy_utils "github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/common/utils/daisy"
 	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/daisycommon"
+	"github.com/GoogleCloudPlatform/compute-image-tools/daisy"
+	daisyCompute "github.com/GoogleCloudPlatform/compute-image-tools/daisy/compute"
+	"google.golang.org/api/compute/v1"
 )
 
 type bootableDiskProcessor struct {
-	workflow        *daisy.Workflow
-	userLabels      map[string]string
-	storageLocation string
-	uefiCompatible  bool
-	noExternalIP    bool
-	network         string
-	OS              string
+	ImportArguments
```

Comment:
Since this is basically inheritance in GoLang, it's a bit strange to have bootableDiskProcessor inherit from ImportArguments